app/src/main/java/org/mozilla/fenix/settings/address/ext/Address.kt

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.address.ext
+
+import mozilla.components.concept.storage.Address
+
+/**
+ * Generate a label from a given [Address]. A well-formed [Address] will have
+ * a full name in the label, otherwise it will find the highest priority [Address] property
+ * to use, as based on the desktop code found in: FormAutofillUtils::getAddressLabel
```

Comment:
`FormAutofillUtils::getAddressLabel` doesn't tell us what they do with the name so, let's find out where that is done and validate our approach

Reply:
Just testing it manually it looks like it does allow for `middleName lastName`, so I'll simplify my implementation to match